Automattic/jetpack
# VideoPress: Enable quick actions on video list for mobile

Fixes #26531

#### Changes proposed in this Pull Request:

* Added quick actions for videos on list layout for mobile

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard on mobile
* If there are no videos, upload one
* Change the layout to list instead of grid
* Click on the video
* Check that there are quick actions available

Before | After
---------|-------
![vp-list-actions](https://user-images.githubusercontent.com/8486249/193113148-1ddc73e7-f158-47dd-b614-c11785c8dbf2.jpg) | ![vp-list-actions-after](https://user-images.githubusercontent.com/8486249/193113142-81d2be8a-ae2c-47d8-988a-7ed5a6e34587.jpg)
